### ISSUE TITLE
Connect GraphQL

### ISSUE THREAD

**Issue by saulyz** [pets-oss/pets-front#22]
Connect an Apollo client, create GraphQL wrapper (ApolloProvider) for app.
Create demo queries for animal listing (use page Animal-list), use docs and backend endpoint - https://petbook-back.herokuapp.com/graphiql 

No styling is needed in this task (proper UI and styles - is another task).
Consider GQL query organisation in some meaningful manner. Probably use `graphql-tag`.
Could we make a configurable GQL endpoint (like .env for certain deployment)?

This task depends on #4  and route 'animal-list' should be working.

Some reads: 

- https://www.sitepoint.com/how-to-build-a-web-app-with-graphql-and-react/
- https://blog.logrocket.com/build-a-graphql-react-app-with-typescript/
